### PR TITLE
simplify package.json

### PR DESCRIPTION
**Pull request by sindresorhus**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,15 +2,9 @@
...
-  "bin": {
-    "lighthouse": "cli.js"
-  },
-  "directories": {
-    "test": "tests"
-  },
+  "bin": "cli.js",
   "engines": {
-    "node": ">=4.0.0"
+    "node": ">=5"
```
Comment (sindresorhus) [inline package.json]: You use Node.js 5 features in this package.

File: package.json [hunk 2]
```diff
@@ -2,15 +2,9 @@
   "name": "lighthouse",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
-  "bin": {
-    "lighthouse": "cli.js"
-  },
-  "directories": {
-    "test": "tests"
-  },
```
Comment (sindresorhus) [inline package.json]: `directories` is not used by npm at all.

File: package.json [hunk 3]
```diff
@@ -33,7 +28,6 @@
     "chrome-devtools-frontend": "1.0.381789",
     "chrome-remote-interface": "^0.11.0",
     "devtools-timeline-model": "1.0.16",
-    "file": "^0.2.2",
```
Comment (sindresorhus) [inline package.json]: This dependency isn't used.

**Comment by paulirish** [comment]
thx man!

**Pull request merged**